transports/webrtc/src/tokio/connection.rs

```diff
@@ -144,6 +151,22 @@ impl Connection {
             })
         }));
     }
+
+    /// Returns the total number of bytes received by all connections.
```

Comment:
don't like that API exposes a function which don't belong here ("all connections").